Modules/_lsprof.c

```diff
@@ -750,7 +750,7 @@ profiler_dealloc(ProfilerObject *op)
     if (op->flags & POF_ENABLED) {
         PyThreadState *tstate = _PyThreadState_GET();
         if (_PyEval_SetProfile(tstate, NULL, NULL) < 0) {
-            PyErr_WriteUnraisable((PyObject *)op);
+            _PyErr_WriteUnraisableMsg("When destroying _lsprof profiler", NULL);
```

Comment:
Interestigly, I catched this bug with the change: the object here is dead so is not correct to pass it to the unraisable hook.